File: konzolos/CarsConsole/Booking.cs
using System;

namespace CarsConsole.Models
{
    public class Booking
    {
        public int BookingId   { get; set; }
        public int CarId       { get; set; }
        public DateTime StartDate { get; set; }
        public DateTime EndDate   { get; set; }

        public int Days => (EndDate - StartDate).Days + 1;
        public decimal TotalPrice => Days * DailyPrice;

        // Ezt a DailyPrice-t és a Brand/Model-t a LoadBookings állítja be
        public decimal DailyPrice { get; set; }

        // Új mezők:
        public string Brand { get; set; } = string.Empty;
        public string Model { get; set; } = string.Empty;
    }
}

File: app/Models/Booking.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Model;
use Carbon\Carbon;

class Booking extends Model
{
    protected $fillable = [
        'car_id',
        'start_date',
        'end_date',
    ];

    // automatikusan csatoljuk a számított mezőt
    protected $appends = ['total_price'];

    public function car()
    {
        return $this->belongsTo(Car::class);
    }

    // napi díj * napok száma
    public function getTotalPriceAttribute()
    {
        $start = Carbon::parse($this->start_date);
        $end   = Carbon::parse($this->end_date);
        $days  = $start->diffInDays($end);
        return $days * $this->car->daily_price;
    }
}